src/repositories/in-memory-db/inMemoryProjectRepository.ts

```diff
@@ -0,0 +1,29 @@
+import { randomUUID } from 'crypto'
+
+import { Prisma, Project } from '@prisma/client'
+import { ProjectRepository } from '../project-repository'
+
+export class InMemoryProjectRepository implements ProjectRepository {
+  public dbProject: Project[] = []
+
+  constructor() {}
+
+  async create(
+    data: Prisma.ProjectCreateWithoutUserInput & { user_id: string },
```

Comment:
it's better if you use the same data type that you're using `Prisma.ProjectUncheckedCreateInput`